anta/models.py

```diff
@@ -140,10 +140,10 @@ class AntaCommand(BaseModel):
         version: eAPI version - valid values are 1 or "latest".
         revision: eAPI revision of the command. Valid values are 1 to 99. Revision has precedence over version.
         ofmt: eAPI output - json or text.
-        output: Output of the command populated by the collect() function
-        template: AntaTemplate object used to render this command
-        params: Pydantic Model containing the variables values used to render the template
-        errors: If the command execution fails, eAPI returns a list of strings detailing the error
+        output: Output of the command. Only defined if there was no errors.
+        template: AntaTemplate object used to render this command.
+        errors: If the command execution fails, eAPI returns a list of strings detailing the error.
+        params: Pydantic Model containing the variables values used to render the template.
```

Comment:
Hmmm as this PR been rebased?

Reply:
Yes

Reply:
Why ?

Reply:
probably you changed teh description but it is not clear here in my diff for params